src/contexts/BreakpointContext.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+export type Breakpoint = 0 | 1 | 2 | 3;
```

Comment:
Same like the rest, we can probably make it a bit more dynamic, so maybe a plain number is fine.